### ISSUE TITLE
Meteor 2.5.3 fibers build fails

### ISSUE THREAD

**Issue by vjau** [meteor/meteor#11861]
I can't install my Meteor app upgraded at **2.5.3** on my Ubuntu 20.04 server.
`sudo npm i `in programs/server gives me errors related to the fibers build.
Downgrading to **version 2.5** builds successfuly. 

Here is the error i get in the npm log 

```
554 verbose stack Error: fibers@https://github.com/meteor/node-fibers/archive/refs/tags/5.0.0-1.tar.gz install: `node build.js || nodejs build.js`
554 verbose stack spawn ENOENT
554 verbose stack     at ChildProcess.<anonymous> (/usr/lib/node_modules/npm/node_modules/npm-lifecycle/lib/spawn.js:48:18)
554 verbose stack     at ChildProcess.emit (events.js:400:28)
554 verbose stack     at maybeClose (internal/child_process.js:1058:16)
554 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:293:5)
555 verbose pkgid fibers@https://github.com/meteor/node-fibers/archive/refs/tags/5.0.0-1.tar.gz
556 verbose cwd /var/local/app/compta/bundle/programs/server
557 verbose Linux 5.4.0-94-generic
558 verbose argv "/usr/bin/node" "/usr/bin/npm" "install"
559 verbose node v14.18.3
560 verbose npm  v6.14.15
561 error code ELIFECYCLE
562 error syscall spawn
563 error file sh
564 error errno ENOENT
565 error fibers@https://github.com/meteor/node-fibers/archive/refs/tags/5.0.0-1.tar.gz install: `node build.js || nodejs build.js`
565 error spawn ENOENT
566 error Failed at the fibers@https://github.com/meteor/node-fibers/archive/refs/tags/5.0.0-1.tar.gz install script.
566 error This is probably not a problem with npm. There is likely additional logging output above.
567 verbose exit [ 1, true ]
```
Thank you.

**Comment by pmogollons** [comment]
I think this is a duplicate of #11791 which is already fixed.

You should be able to update to 2.5.6 without any issues.

**Comment by pmogollons** [comment]
Also to note, this is fixed in 2.5.6 but not on 2.6-rc.0

**Comment by renanccastro** [comment]
This is fixed on 2.5.6 as @pmogollons correctly noted! Thanks!
